test/unit/report_importer_test.rb

```diff
@@ -10,7 +10,7 @@ class ReportImporterTest < ActiveSupport::TestCase
   end
 
   test 'it should import reports with no metrics' do
-    r = ReportImporter.import(read_json_fixture('report-empty.json'))
+    r = ConfigReportImporter.import(read_json_fixture('report-empty.json'))
```

Comment:
Right, but this ought to be a behaviour of the ReportImporter too, as it led the error I reported in https://github.com/theforeman/foreman/pull/2750#issuecomment-147375997 and that you changed the #metrics method to handle.  It should be tested and functional on ReportImporter.  I suspect at the moment it only works for ConfigReportImporters, as per https://github.com/theforeman/foreman/pull/2750#discussion-diff-42617415